janus_core/md.py

```diff
@@ -657,6 +708,62 @@ def __init__(
             externalstress=self.pressure * units.bar,
         )
 
+    @property
+    def _parameter_prefix(self) -> str:
+        """
+        Ensemble parameters for output files.
+
+        Returns
+        -------
+        str
+           Formatted temperature range if heating and target temp if running md.
+        """
+
+        temperature_prefix = ""
+        if self.temp_start and self.temp_end:
+            temperature_prefix = f"-T{self.temp_start}-T{self.temp_end}"
+
+        if self.steps > 0:
+            temperature_prefix = f"{temperature_prefix}-T{self.temp}"
+
+        pressure = f"-p{self.pressure}" if not isinstance(self, NVT_NH) else ""
+        return f"{pressure}{temperature_prefix}"
```

Comment:
Not particularly important, but is there a benefit in swapping pressure and temperature from the original ordering?

Could we not also call the base function to get the temperature prefix, and append (or prepend) the pressure here?

Reply:
Honestly I have gone through a lot of revisions of this code, it may no longer make sense to permute. 

I *think* it is a hangover from when the pressure was passed in as a variable? Maybe.

We could have the base handle the temperature yes

Reply:
@ElliottKasoar  The pressure is now restored as after the temperature, and we now use the base to get the temperature